lib/linear_gauge/custom_label/custom_linear_gauge_label.dart

```diff
@@ -0,0 +1,40 @@
+class CustomLinearGaugeLabel {
```

Comment:
I think we need a small change here 
can we change the class name to `RulerCustomLabel`? I think that make more sense .
What do you think ?

Reply:
Yes agreed that will make more sense . Custom labels are for all gauges not only for linear. Will change that.

Reply:
Or how about maybe CustomRulerLabel? Lets try to keep RulerLabel together

Reply:
As per terminology we used , labels are part of rulers only. So `CustomRulerBar` or `CustomRuler` can also work but for user `CustomRulerLabel` make more sence . So what should be the final call